lib/xnvme_be_cbi_sync_psync.c

```diff
@@ -13,7 +13,6 @@
 #ifdef XNVME_BE_CBI_SYNC_PSYNC_ENABLED
 #include <errno.h>
 #include <unistd.h>
-#include <libxnvme_spec_fs.h>
```

Comment:
This should include libxnvme.h

Reply:
Since the internal headers still include other headers, then it gets ``libxnvme.h`` via ``xnvme_be.h``